Gui/opensim/CMakeLists.txt

```diff
@@ -16,6 +16,22 @@ endif()
 
 find_package(Ant REQUIRED)
 
+# http://xit0.org/2013/04/cmake-use-git-branch-and-commit-details-in-project/
+# Get the latest abbreviated commit hash of the working branch
+execute_process(
+  COMMAND git log -1 --format=%h
```

Comment:
Not everyone will have `git` on their path. See https://cmake.org/cmake/help/latest/module/FindGit.html

It would be good to use something like the following:

```cmake
find_package(Git)
if(NOT Git_FOUND)
  set(GIT_COMMIT_HASH "nocommit")
else()
  <what you already have>
endif()
```

Reply:
👍